sideboard/internal/logging.py

```diff
@@ -21,25 +25,25 @@ def _configure_logging():
     if os.path.exists(fname):
         logging.config.fileConfig(fname, disable_existing_loggers=True)
     else:
-        format_str = '%(asctime)s [%(levelname)s] %(name)s: %(message)s'
+        # ConfigObj doesn't support interpolation escaping, so we manually work around it here
+        formatters = config['formatters'].dict()
```

Comment:
The other changes in this file are just to add custom formatter support (mentioned above in my comment in ``sideboard/configspec.ini``).